ingest/expression_files/mtx.py

```diff
@@ -116,6 +143,27 @@ def check_duplicates(names: List, file_type: str):
             raise ValueError(msg)
         return True
 
+    @staticmethod
+    def get_line_no(file_handler: str) -> int:
```

Comment:
I don't understand this argument -- it's called file_path in the above method, and file_handler here.  Is it just the string path?  or is it a file handler?  
 The method should be renamed more descriptively.  what line number are we getting?  Maybe 'get_data_start_line_number'?    This function is only called a couple of times so no need to be concise

Reply:
5e604cd